db/migrate/20180122193043_create_wizards.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWizards < ActiveRecord::Migration[5.1]
+  def change
+    create_table :wizards do |t|
+      t.string :name, null: false, index: true
+      t.string :img_url
+      t.text :description, null: false
+      t.integer :created_by
```

Comment:
If this is for an association with the users table, I would suggest making it an `id` column, like so: `creator_id`